lms/views/__init__.py

```diff
@@ -5,6 +5,8 @@
     CanvasStudentNotInGroup,
 )
 
+from lms.views.api.blackboard.exceptions import BlackboardStudentNotInGroup
+
```

Comment:
I think this import is unnecessary since `BlackboardStudentNotInGroup` isn't used outside of the `lms.views.api.blackboard` package. The reason why we do this in packages like `models`, `validation`, `services`, etc is because it makes it easier for other packages to import things. Instead of:

```python
from lms.models.user import User
from lms.models.grouping import CanvasGroup
from lms.models.application_instance import ApplicationInstance
```

You can just do:

```python
from lms.models import User, CanvasGroup, ApplicationInstance
```

It also means that the internal structure of a package like `lms.models` is effectively a private implementation detail of that package which is nice because other packages don't need to know about it (hence imports are easier to write) and because we can reorganise the code inside `lms.models` without breaking any other packages. There's good reason to split a package into separate modules (we don't want one really big file) but also good reason not to tightly couple other packages to that sub-organization.

In this case `BlackboardStudentNotInGroup` isn't being exported for other packages to import so there's no need.

(The imports of the Canvas exceptions above should probably be removed for the same reason.)